packages/typespec-azure-resource-manager/lib/responses.tsp

```diff
@@ -138,7 +138,7 @@ model ArmOperationStatus<
 model ArmAsyncOperationHeader<
   StatusMonitor extends {} = ArmOperationStatus,
   UrlValue extends string = string,
-  FinalResult extends {} = never
+  FinalResult extends {} | void = never
```

Comment:
Is void resulting in different than never

Reply:
Yes,  never causes the decorator not to execute; void has it execute with value void.  The difference in the LRO metadata is subtle, but important

Reply:
In terms of the lro, never means the last call is to the polling endpoint,  'void'  means the endpoint is dereferenced again after polling completes and returns an empty body response